ptvsd/wrapper.py

```diff
@@ -1509,7 +1521,8 @@ def on_stackTrace(self, request, args):
             key = (pyd_tid, int(xframe['id']))
             fid = self.frame_map.to_vscode(key, autogen=True)
             name = unquote(xframe['name'])
-            norm_path = self.path_casing.un_normcase(unquote(xframe['file']))
+            # pydevd encodes if necessary and then uses urllib.quote.
+            norm_path = self.path_casing.un_normcase(unquote(str(xframe['file'])))
```

Comment:
This should be fine as long as `xframe['file']` is never `bytes` in Python 3, which won't be a problem.

Reply:
Yes that's the case, its not a problem.